spec/services/status_page_spec.rb

```diff
@@ -13,10 +13,11 @@
         }.to_json
       )
     end
+    let(:redis) { Redis.new(host: 'localhost', post: 6379, db: 0) }
```

Comment:
```suggestion
    let(:redis) { described_class.new.send(:redis) }
```

ça sera plus maintenable